src/main/resources/templates/report.html

```diff
@@ -27,6 +27,9 @@ <h1 class="page-header" th:text="${assignmentId} + ' submissions'"></h1>
         <li role="presentation" th:classappend="${mode == 'testMatrix'} ? 'active' : ''">
             <a th:href="@{'/testMatrix/' + ${assignmentId}}" data-toggle="tab" data-target="#myTab">Test Matrix</a>
         </li>
+        <li role="presentation" th:classappend="${mode == 'signalledSubmissions'} ? 'active' : ''">
+            <a th:href="@{'/cenas/' + ${assignmentId}}" data-toggle="tab" data-target="#myTab">Signalled Students</a>
```

Comment:
Please change "cenas" to "signalledSubmissions" or something... :)

Reply:
Done